kuma/static/js/lang-switcher.js

```diff
@@ -3,7 +3,19 @@
     var neverShowNoticeKey = 'never-show-locale-notice';
     var neverShowNotice = getNeverShowNotice();
     function storeLocaleChange(code, name) {
-        sessionStorage.setItem(sessionStorageKey, JSON.stringify({code: code, name: name}));
+        if (!isExistingLocaleCookieExist(code)){
+            sessionStorage.setItem(sessionStorageKey, JSON.stringify({code: code, name: name}));
+        }
+    }
+
+    function getCookie(name) {
+	    var v = document.cookie.match('(^|;) ?' + name + '=([^;]*)(;|$)');
+	    return v ? v[2] : null;
+	}
+
+    function isExistingLocaleCookieExist(code) {
```

Comment:
Nit: Would this read better as `localeCookieExist`?

Reply:
I agree, the name of the function doesn't seem to match the purpose. ``isLocalePreference`` seems better to me - ``true`` if ``code`` is the user's current locale preference override, ``false`` if ``code`` is not the user's locale preference or if no preference is set.